src/main/webapp/app/views/site-data-view/site-data.service.js

```diff
@@ -2,14 +2,10 @@
     'use strict';
     angular
         .module('vivaxDataManagerApp')
-        .factory('SiteDataViewDTO', SiteDataViewDTO);
-
-    SiteDataViewDTO.$inject = ['$resource'];
-
-    function SiteDataViewDTO ($resource) {
-    	var resourceUrl =  'api/siteDataView/:id';
-
-        return $resource(resourceUrl, {}, {
+        .factory('SiteDataViewDTO', function($resource) {
+            var data = [{name:'country', query:'China'}];
+               //return $resource('api/siteData/searchByFilters');
+               return $resource('api/siteData/searchByFilters', {}, {
```

Comment:
In this case it would be better to use %http.post instead of resource because you are not going to create CRUD operations, you only are gonna access one service which is searchByFilters